server/email-content-parser.test.ts

```diff
@@ -121,9 +121,138 @@ To remove your email address permanently from future mailings, please click
 here.
 `;
 
-    const cleaned = cleanEmailContent(userForwardedEmail);
+    const cleaned = parseEmailBody(userForwardedEmail);
     expect(cleaned).toBe(
       "Hi David\n\nI=E2m reaching out about an exciting Full Stack Engineer opportunity with a\nfast-growing Series B SaaS company. They=E2s scaling quickly and investing\nheavily into their engineering practice, having doubled the headcount size\nin the last month alone.\n\nHere=E2s what=E2s on offer:\n\n- **Where:** Remote based role\n- **Package:** Up to $200k base salary + meaningful equity, unlimited PTO\nand more\n- **Tech Stack:** TypeScript, React, Node.js, AWS, Postgres\n\nThe work is focused on addressing major inefficiencies in the construction\nindustry =E2 giving you the chance to build solutions with real-worlds\nimpact.\n\nWould you be open to a quick chat to explore this role further? If =E2s not\nquite what you=E2re looking for, I=E2d still love to hear what=E2s next on your\nradar so I can keep you in mind for future opportunities.\n\n[Image: Planet Shine]\n\n*John Smith*\nSenior Recruiter, *Example Corp*\n\nAI/ML & Software Recruitment Specialist\n+1 646-298-3569 • 15464293569•\nwww.example.com"
     );
   });
+
+  describe("parseEmailBody MIME parsing", () => {
+    it("should extract text/plain content from multipart email", () => {
+      const multipartEmail = `Content-Type: multipart/alternative; boundary="boundary123"
+
+--boundary123
+Content-Type: text/plain; charset="UTF-8"
+
+Apply to this job for me
+
+---------- Forwarded message ---------
+From: James Banner <james@zestfortech.com>
+Date: Fri, Sep 12, 2025 at 2:17 PM
+Subject: Scaling start up - Expanding their Full Stack Engineering team.
+To: <dvargas92495@gmail.com>
+
+Hi David
+I'm reaching out about an exciting Full Stack Engineer opportunity.
+--boundary123
+Content-Type: text/html; charset="UTF-8"
+
+<p>Apply to this job for me</p>
+<p>HTML version content</p>
+--boundary123--`;
+
+      const result = parseEmailBody(multipartEmail);
+      expect(result).toContain("Apply to this job for me");
+      expect(result).toContain("Hi David");
+      expect(result).not.toContain("<p>");
```

Comment:
For all of these tests, have a single assertion that expects the whole output matches an expected full string